src/snippets/responsive-image.liquid

```diff
@@ -12,6 +12,8 @@
     - max_height: {Number} Max height of the image container
     - img_class: {String} class name of the <img />
     - wrapper_class: {String} class name of the <div> wrapper
+    - image_attributes {String} <img> optional HTML attributes
+    - wrapper_attributes {String} <div> wrapper optional HTML attributes
```

Comment:
Please change the wording to match the structure above. Suggestion:

```
- image_attributes: {String}  additional HTML attributes of the <img /> 
- wrapper_attributes: {String} additional HTML attributes of the <div> wrapper
```

Also, could you please group these `img_` and `wrapper_` concerns together?

```
- img_class:
- image_attributes:
- wrapper_class:
- wrapper_attributes:
```

Reply:
On the same note, can we also have consistency between `img` and `image`? Personal preference: `image_class` and `image_attributes`.